articles/protocols/saml/saml-configuration/special-configuration-scenarios/index.md

```diff
@@ -0,0 +1,19 @@
+---
+  description: Special configuration scenarios when setting up a SAML Integration
+  url: /protocols/saml/saml-configuration/special-configuration-scenarios
```

Comment:
I think that the `url` metadata is not needed. Even without it the `index` part is getting ignored. You should cross check though.